test/unit/utils/index.spec.ts

```diff
@@ -34,6 +34,23 @@ interface Obj {
   [key: string]: any;
 }
 
+describe('SDK_VERSION', () => {
+  it('utils index should retrieve the SDK_VERSION from package.json', () => {
+    const mockVersionNumber = "MockVersion";
```

Comment:
May be this test case can just read the version from `../../../package.json` and compare that to what's returned by `utils`.